iphone/CoreApi/CoreApi.xcodeproj/project.pbxproj

```diff
@@ -895,8 +893,6 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				479834F023426C0100724D1E /* common-debug.xcconfig in Resources */,
-				479834F123426C0100724D1E /* common-release.xcconfig in Resources */,
```

Comment:
Hm, really not needed here?

Reply:
I have no idea. I've changed the project by using Xcode UI. I will try to revert this change and see what happens.

Reply:
Reverted